taxonomy-editor-frontend/src/pages/editentry/index.tsx

```diff
@@ -119,4 +126,30 @@ const EditEntry = ({ addNavLinks }) => {
   );
 };
 
-export default EditEntry;
+type EditEntryWrapperProps = {
+  addNavLinks: ({
+    branchName,
+    taxonomyName,
+  }: {
+    branchName: string;
+    taxonomyName: string;
+  }) => void;
+};
+
+const EditEntryWrapper = ({ addNavLinks }: EditEntryWrapperProps) => {
```

Comment:
Why is there a wrapper component here, btw? Any specific reason we can't do these checks inside `EditEntry`?

Reply:
it is because the params can be `string` or `undefined`, and as these are used in several places I consider it is better not to have to check every time but just consume it. The wrapper is only enforcing the type to be a string.